README.md

```diff
@@ -1,18 +1,34 @@
+![plexupdate.sh](http://i.imgur.com/ThY5Rvl.png "plexupdate")
 # plexupdate
 
-Plex Update is a BASH script which simplifies the life of headless Linux Plex Media Server users (how's that for a strange description).
+Plex Update is a BASH script which simplifies the life of headless Linux Plex Media Server users (what's that for a strange description).
```

Comment:
Actually, "How's that for a ...." is an expression and is correct. Changing it to what's that... does not make any sense. See https://www.google.com/search?q=%22how+is+that+for+a%22&oq=%22how+is+that+for+a%22